util/make_new_dif/templates/dif_autogen.c.tpl

```diff
@@ -325,6 +325,30 @@ dif_result_t dif_${ip.name_snake}_init(
     return kDifOk;
   }
 
+  OT_WARN_UNUSED_RESULT
```

Comment:
I wonder if we should rename and slightly modify the implementation of the existing `dif_dev_irq_force(dev, irq)` function to expose `dif_dev_irq_set(dev, irq, val {true, false})`. @timothytrippel WDYT?

Reply:
yeah that make sense to me! It would simplify the interface, but maybe: `dif_<dev>_irq_force_set(<ip>, irq, value)`  ? (to be consistent with existing terminology)